server/games/game.py

```diff
@@ -186,6 +189,45 @@ def teams(self):
         return frozenset({self.get_player_option(player.id, 'Team')
                           for player in self.players})
 
+    def team_count(self):
+        teams = defaultdict(int)
+        for player in self.players:
+            teams[self.get_player_option(player.id, 'Team')] += 1
+
+        return teams
+
+    def even_teams(self):
```

Comment:
For a predicate-like function I'd prefer to make it clear -- consider naming it just `is_even`? This would be consistent with your predicate-like `is_ffa` as well.

Perhaps even cleaner for the callsites would be to make these getter functions, so you can merely write `game.is_even` or `game.is_ffa`.